File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-26.cs
public class Solution {
    public int EvalRPN(string[] tokens) {
        var res = 0;
        var stack = new Stack<int>();
        for (int i = 0; i < tokens.Length; i++) {
            var c = tokens[i];
            if (c == "+") {
                stack.Push(stack.Pop() + stack.Pop());
            }
            else if (c == "-") {
                var a = stack.Pop();
                var b = stack.Pop();
                stack.Push(b - a);
            }
            else if (c == "*") {
                stack.Push(stack.Pop() * stack.Pop());
            }
            else if (c == "/") {
                var a = stack.Pop();
                var b = stack.Pop();
                stack.Push((int)Math.Truncate((double)b / a));
            }
            else {
                stack.Push(int.Parse(c));
            }
        }
        return stack.Peek();
    }
}

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-26.py
class Solution:
    def findMin(self, nums: List[int]) -> int:
        n = len(nums)
        l, r = 0, n - 1
        res = nums[l]
        while l <= r:
            m = l + ((r - l) // 2)
            res = min(nums[m], res)
            if nums[l] > nums[m]:
                r = m - 1
            else:
                if nums[m] < nums[r]:
                    r = m - 1
                else:
                    l = m + 1
        return res